swagger.yaml

```diff
@@ -532,6 +532,119 @@ paths:
         "default":
           description: Software entry not found
 
+  "/questionnaire":
+    get:
+      operationId: listQuestionnaires
+      summary: Get all questionnaire entries
+      description: Retrieve a list of all questionnaire entries in the registry.
+      tags:
+        - Questionnaire
+      responses:
+        "200":
+          description: A list of questionnaire entries
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/Questionnaire"
+
+    post:
+      operationId: createQuestionnaire
+      summary: Register new questionnaire
+      description: Register a new questionnaire entry in the registry.
```

Comment:
I think we can just refer to Questionnaire or questionnaires rather than questionnaire entry. Please remove all references to entry/ entries- if you agree.